nym-vpn-core/Cargo.toml

```diff
@@ -103,22 +104,23 @@ zeroize = "1.6.0"
 
 # Use the develop-pre-ecash-for-vpn branch until we have ported the vpn client to use ecash
 
-nym-authenticator-requests = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-bandwidth-controller = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-bin-common = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-client-core = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-config = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-credential-storage = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-credentials = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-crypto = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-explorer-client = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-http-api-client = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-id = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-ip-packet-requests = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-node-requests = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-pemstore = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-sdk = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-task = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-topology = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-validator-client = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
-nym-wireguard-types = { git = "https://github.com/nymtech/nym", rev = "3f922cc0" }
+nym-authenticator-requests = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+nym-bin-common = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+nym-client-core = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+nym-config = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+nym-crypto = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+nym-explorer-client = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+nym-http-api-client = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+nym-ip-packet-requests = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+nym-node-requests = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+nym-pemstore = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+nym-sdk = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+nym-task = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+nym-topology = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+nym-validator-client = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+nym-wireguard-types = { git = "https://github.com/nymtech/nym", rev = "dff82f9" }
+
+nym-credential-storage-fork = { git = "https://github.com/nymtech/nym", rev = "3f922cc0", package = "nym-credential-storage" }
```

Comment:
rather than calling it `fork`, maybe explicitly say it's coconut/pre-ecash/something to indicate why it was forked?